src/components/Agencies/Agencies.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Agencies = () => {
+  return <div>Agencies</div>;
+};
```

Comment:
```suggestion
const Agencies = () => (
  <div>Agencies</div>;
);
```

Reply:
Please this code style everywhere if component consist of one element

Reply:
Done